keepsorted/options.go

```diff
@@ -99,7 +99,7 @@ func (f *Fixer) parseBlockOptions(startLine string) (blockOptions, error) {
 
 		val, err := parseBlockOption(field, startLine)
 		if err != nil {
-			errs = multierr.Append(errs, err)
+			errs = errors.Join(errs, err)
```

Comment:
there is a bit of a semantic difference here where multierr appends into the same error while errors.Join keeps creating new errors (flat list vs linked list)...but that should be fine for now. We can always change it later if we need to

no action necessary, just noting this in case someone tries to figure out why we changed the semantics here